deploy/kubeturbo-operator/deploy/crds/charts_v1alpha1_kubeturbo_cr.yaml

```diff
@@ -33,8 +33,8 @@ spec:
 
   # Uncomment next lines to specify a repository and image tag for kubeturbo
   #image:
-  #  repository: registry.connect.redhat.com
-  #  tag: 8.4.0
+  #  repository: icr.io/cpopen/turbonomic/kubeturbo
+  #  tag: 8.7.3
   # Uncomment to use an image from RHCC for cpu-frequency getter job - predefined in OCP Operator Hub version
   #  busyboxRepository: registry.access.redhat.com/ubi8/ubi-minimal
```

Comment:
What is our default `busyboxRepository` now?

Reply:
-->$skopeo list-tags docker://icr.io/cpopen/turbonomic/busyboxturbo
{
    "Repository": "icr.io/cpopen/turbonomic/busyboxturbo",
    "Tags": [
        "1.0"
    ]
}

Reply:
Busybox work is postponed after discussion.